### PR TITLE
docs(install): add Termux workaround for search

### LINKED ISSUE #1730: update Termux docs
The new version can't run, DEP can't install incompatible system, code package can't run, can't create / usr / local / bin directory, I don't have root permission, I hope the author can change the code package directory to the current user's home directory.
新版的无法运行,dep无法安装不兼容我的系统，code 包无法运行无法创建/usr/local/bin目录 我没有root权限，希望作者能把code包目录改为当前用户家目录.
I'm crazy.

$ ./code-server
./code-server: 20: exec: /data/data/com.termux/files/home/code-server-3.3.1-linux-arm64/bin/../lib/node: not found

https://github.com/cdr/code-server/issues/1685
$ mkdir -p /usr/local/bin/
mkdir: cannot create directory ‘/usr’: Read-only file system

$ dpkg -i code-server_3.3.1_arm64.deb
dpkg: error processing archive code-server_3.3.1_arm64.deb (--install):
 package architecture (arm64) does not match system (aarch64)
Errors were encountered while processing:
 code-server_3.3.1_arm64.deb

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR updates the `install.md` docs with a workaround for the search issue on Android. 

H/T to @yozman for posting their solution: https://github.com/cdr/code-server/issues/1730#issuecomment-721515979

Fixes #1730

### CODE REVIEW

**Comment by dudss37** [comment]
[]()

File: docs/install.md [hunk 1]
```diff
@@ -152,7 +168,7 @@ yarn global add code-server
 # Or: npm install -g code-server
 code-server
 # Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
-```
+````
```
Comment (oxy) [inline docs/install.md:171]: There's an extra backquote here that I think maybe shouldn't be here? - surprised it went through `yarn fmt` without complaining actually :eyes:

Comment (jsjoeio) [reply docs/install.md:171]: whoops!

**Review by jawnsy — APPROVED** [review]
looks awesome to me!

File: docs/install.md [hunk 2]
```diff
@@ -134,6 +135,23 @@ Termux is an Android terminal application and Linux environment, which can also
...
 
+### Known Search Issue
+
+There is a known issue with search not working on Android because it's missing `bin/rg`. To fix:
+
+1. Install `ripgrep` with `pkg`
+   ```sh
+   pkg install ripgrep
+   ```
+2. Make a soft link using `ln -s`
```
Comment (jawnsy) [inline docs/install.md:146]: doesn't matter at all, but using autonumbering might be useful?

Comment (jsjoeio) [reply docs/install.md:146]: Assuming you're referring to the 2 steps i.e. 
1.
1.

Instead of 
1. 
2.

? I think it's fine!

Comment (jawnsy) [reply docs/install.md:146]: yea, and I agree it's fine as it is :)

**Pull request merged**